openshift/e2e-tests-openshift.sh

```diff
@@ -6,14 +6,14 @@ set -e
 
 # Maximin number of parallel tasks run at the same time
 # # start from 0 so 4 => 5
-MAX_NUMBERS_OF_PARALLEL_TASKS=4
+MAX_NUMBERS_OF_PARALLEL_TASKS=1
```

Comment:
Is that what you want to do ? We had running as 0 before, and it was quite slow (would timeout the openshift-ci job), is 2 a good number ?

Reply:
this is just for testing :sweat_smile: 
I have skipped `non-privileged` and just wanted to test the `privileged` ones. Also under privileged section I have put `orka-*`  tasks so testing 2 at a time.

Reply:
with this --> `# test_non_privileged $(\ls -1 -d task/*/*/tests)` I guess all the non-privileged tasks would not be tested?